pkgs/development/tools/jq/default.nix

```diff
@@ -1,27 +1,83 @@
-{ stdenv, fetchurl, oniguruma }:
+{ stdenv, buildPackages, fetchFromGitHub
+, oniguruma
+# installCheck inputs
+, bundlerApp ? null
+# features
+, enableAsan ? false
+, enableUbsan ? false
+, enableValgrindChecks ? true
```

Comment:
I would rather prefer to not having to maintain these feature flags.
`enableValgrindChecks` since to have no impact on the runtime closure size.
`enableAsan` and `enableUbsan` are mostly useful for development/debugging; users/developers who wish to enable those, can as well add an override to extend `configureFlags`.
This would make the nix expression less complex.

Reply:
Dropping `enableAsan` and `enableUbsan` sounds good. I'd still be for keeping `enableValgrindChecks` despite the bit of added complexity due to the speedup when debugging the derivation?

Reply:
Maybe disable Valgrind checks unconditionally? They also seem resonably easy to override to enabled.